packages/core/src/sync-store/encode.ts

```diff
@@ -33,7 +33,7 @@ export const encodeBlock = ({
   receiptsRoot: block.receiptsRoot,
   transactionsRoot: block.transactionsRoot,
   sha3Uncles: block.sha3Uncles ?? null,
-  size: block.size ? hexToBigInt(block.size) : 0n,
+  size: hexToBigInt(block.size),
```

Comment:
Now that we're setting the default values in the "standardize" functions, there are probably more spots in this file we can simplify right?

Fine to do as a follow-up, just checking for understanding.

Reply:
For all the previous cases of RPC not providing all properties before `block.size`, we would just mark the column as nullable.

https://github.com/ponder-sh/ponder/blob/acee34601710394e0e2094e35c9c0a4757042c2f/packages/core/src/sync-store/migrations.ts#L442-L457